src/stream/src/executor/backfill/snapshot_backfill/executor.rs

```diff
@@ -209,8 +209,9 @@ impl<S: StateStore> SnapshotBackfillExecutor<S> {
 
                     let recv_barrier = self.barrier_rx.recv().await.expect("should exist");
                     assert_eq!(first_upstream_barrier.epoch, recv_barrier.epoch);
-                    backfill_state.commit(recv_barrier.epoch).await?;
+                    let post_commit = backfill_state.commit(recv_barrier.epoch).await?;
                     yield Message::Barrier(recv_barrier);
+                    post_commit.post_yield_barrier(None).await?;
```

Comment:
Is it possible to introduce a sugar method for `post_yield_barrier(None)`?

Reply:
In most cases, if we are sure that there is no update vnode bitmap, we should call `state_table.commit_assert_no_update_vnode_bitmap`. This special call of `post_yield_barrier(None)` is only a temporary workaround for snapshot backfill executor, which hasn't, but will support update vnode bitmap in the future. The snapshot backfill executor has its own `ProgressState` struct that wraps the `StateTable` and has its own `commit` method. We don't have to provide a separate method sugar `commit_assert_no_update_vnode_bitmap` for this temporary workaround.